CONTRIBUTION.md

```diff
@@ -0,0 +1,184 @@
+Your git signature must match your Eclipse Foundation account information to be accepted.
+If your signature was accepted is visible by checking the status check on your pull request. 
+
+For more information, please see the Eclipse Committer Handbook:
+https://www.eclipse.org/projects/handbook/#resources-commit
+
+> **Note**: You can submit issues without signing the ECA. 
+
+### Contact
+
+Contact the project developers via the project's "dev" list.
+
+* https://dev.eclipse.org/mailman/listinfo/antenna-dev
+
+## How Can I Contribute?
+
+### Before Submitting An Issue
+
+* **Check the [troubleshooting documentation](https://github.com/eclipse/antenna/blob/master/antenna-documentation/src/site/markdown/troubleshooting.md.vm).** 
+You might be able to find the cause of the problem and fix things yourself. Most importantly, check if you can reproduce the problem [in the current version of Antenna](https://github.com/eclipse/antenna/blob/master). 
+It is possible that the bug has already been fixed. 
+* **Check the [documentation](https://github.com/eclipse/antenna/tree/master/antenna-documentation/src/site/markdown)**.
+Your issue might already be mentioned there. 
+* **Perform a [cursory search](https://github.com/eclipse/antenna/issues?utf8=%E2%9C%93&q=is%3Aissue)** to see if the problem has already been reported. 
+If it has *and the issue is still open*, add a comment to the existing issue instead of opening a new one.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you want to report, open a new issue and include a link to the original issue in the body of your new one.
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report.
+Before creating bug reports, please check [the "before-submitting-a-bug" list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. 
+Fill out [the required template](https://github.com/bsinno/osm-antenna/issues/new?labels=bug&template=bug_template.md).
+
+#### How Do I Submit A (Good) Bug Report?
+
+We have a template for a [bug report](https://github.com/eclipse/antenna/issues/new?labels=bug&template=bug_template.md). 
+Please use that by clicking on **new issue** and then choose the option **bug report**.
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+* Give a **clear and descriptive title** that identifies the problem.
+* Give a **summary of the bug**.
+* Describe the **steps to reproduce** the bug. 
+    * Don't hesitate to go into detail.
+    * Everything that helps to reproduce the bug on a local machine is helpful. 
+        * Logs with stack traces
+        * Antenna configuration files
+        * System properties
+        * Code snippets
+        * Links to projects that experienced the problem.
+    * Describe the behaviour you experienced in detail and point out what exactly is the problem with that behaviour. 
```

Comment:
"behaviour" is Britisch English, but I believe it should be American English: "behavior"